### DISCUSSION TITLE
monitoring-node-reboots/

### DISCUSSION THREAD

**Discussion by giscus** [netdata/blog#329] [General]
# monitoring-node-reboots/

How to monitor node reboots?

https://blog.netdata.cloud/monitoring-node-reboots/

**Comment by thomasbeaudry** [comment]
This is great, can I ask why the alarm isn't already in the codebase?  It should definitely be added to it.

**Reply by andrewm4894** [reply]
@sashwathn do you know if this is in the defaults or not?

**Comment by sashwathn** [comment]
@thomasbeaudry @andrewm4894 : This alert is not defined by default. The reason being the ephemeral nodes as these will not work efficiently in this case.

@MrZammler : Any suggestions on a better node reboot alert?
